lib/elementWrapper/elementWrapper.js

```diff
@@ -66,6 +66,11 @@ class ElementWrapper {
     const elem = await firstElement.apply(this, [retryInterval, retryTimeout]);
     return await elem.isDraggable();
   }
+
+  async getAttribute(arg) {
```

Comment:
To keep this in sync with other methods like `text`, `value` can we rename this to `attribute`?